### ISSUE TITLE
read_region from level=0 takes too long

### ISSUE THREAD

**Issue by reyvateil** [openslide/openslide-python#100]
## Context

**Issue type** (bug report or feature request): bug
**Operating system** (e.g. Fedora 24, Mac OS 10.11, Windows 10):  Debian 10
**Platform** (e.g. 64-bit x86, 32-bit ARM): x86-64
**OpenSlide Python version** (`openslide.__version__`):  1.1.2
**OpenSlide version** (`openslide.__library_version__`): 3.4.1
**Slide format** (e.g. SVS, NDPI, MRXS): MRXS

## Details

Slides at level=0 are roughly 200,000 x 100,000 pixels. When trying to read_region 512x512 tiles from level=0 every tile takes at least 2min to read. In contrast, when trying to read 512x512 tiles from level=1 (100,000 x 50,000 px) it only takes less than a second.

Is such a jump in extraction time expected?

**Comment by bgilbert** [comment]
No, that isn't expected.  If you're still having this problem, please post enough info to allow us to reproduce this, or at least post the relevant section of your Python code.

**Comment by sj9515** [comment]
Having similar problems a year later. I am opening the slide, reading the region and saving it as a PNG image. Depending on the slide, it takes between 30 seconds to almost 3 minutes at times. Any suggestions?
